README.md

```diff
@@ -1218,6 +1218,64 @@ Direct block HTML can be accessed through `$block['innerHTML']`. This may be use
 
 For another example of how this filter can be used to extend block data, we have implemented a default image block filter in [`src/parser/block-additions/core-image.php`][repo-core-image-block-addition]. This filter is automatically called on `core/image` blocks to add `width` and `height` to image attributes.
 
+---
+
+### `vip_block_data_api__before_parse_post_content`
+
+Modify raw post content before it's parsed by the Block Data API.
```

Comment:
Like you do with the other filter, I might mention that this is the very first thing called so that you are getting exactly what is in the WP database.

I also like the example in the other one, so we could add one here. The example helps people understand why they might use it.

We should also add a warning that manipulating the content directly can cause block parsing errors, and they should be careful with this filter. Here's the language I would use:

> Block Markup is sensitive to changes, even changes in whitespace. We've added this filter to make the plugin flexible, but any changes to the `post_content` should be done with extreme care. Strongly consider adding tests to any filter.

Reply:
Great notes, especially the warning. Will add!

Reply:
@smithjw1 Added an before_parse example along with a warning as you described above. Thank you!